_v2_1/0_sende_post.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Sende post
+identifier: sendepost
+layout: default
+---
+
+For å sende post så må du først lage en `DigitalPost` eller en `FysiskPost`, så opprette en `Forsendelse` og legge ved posten. Deretter sendes denne gjennom en `SikkerDigitalPostKlient`. Når brevet er sendt så kan du spørre om status på en meldingskø. Hvis du mottar en kvittering så kan du sjekke innholdet og så bekrefte mottatt kvittering.
+
+### Opprette digital post
+
+```java
+Sertifikat mottakerSertifikat = null; //Mottakers sertifikat fra Oppslagstjenesten
+String orgnrPostkasse = "123456789";
+
+Mottaker mottaker = Mottaker.builder("99999999999",
+        "ola.nordmann#2222", mottakerSertifikat, orgnrPostkasse).build();
```

Comment:
Hvorfor er `orgnrPostkasse` en egen variabel, mens postkasseadressen ikke er det? Jeg tipper `orgnrPostkasse` er gjort slik for å signalisere at den kommer fra Oppslagstjenesten, men det gjør vel også postkasseadressen?